pkg/apis/application/v1alpha1/types.go

```diff
@@ -381,13 +381,16 @@ type ApplicationSourceKustomize struct {
 	ForceCommonLabels bool `json:"forceCommonLabels,omitempty" protobuf:"bytes,7,opt,name=forceCommonLabels"`
 	// ForceCommonAnnotations specifies whether to force applying common annotations to resources for Kustomize apps
 	ForceCommonAnnotations bool `json:"forceCommonAnnotations,omitempty" protobuf:"bytes,8,opt,name=forceCommonAnnotations"`
+	// Namespace overrides Kustomize namespace
```

Comment:
```suggestion
	// Namespace sets the namespace that Kustomize adds to all resources
```

I think this is slightly more clear?